packages/mdc-base/foundation.js

```diff
@@ -19,21 +19,21 @@
  */
 export default class MDCFoundation {
 
-  /** @return enum{cssClasses} */
+  /** @return {!Object<string>} */
```

Comment:
How come these `enum{...}` return types were changed? They seemed to work fine when we tested them.